### DISCUSSION TITLE
Possibility to "import" data from other forum platform?

### DISCUSSION THREAD

**Discussion by endel** [community/community#36] [Discussions]
Would GitHub provide some API to import entries from other platforms into the Discussions platform?

I'm currently using NodeBB, and I'd be willing to write a tool to export its data into a format that GitHub Discussions could understand and is able to import it from.

Thanks!

**Comment by becca** [comment]
hey @endel thank you for this feedback! we would like to provide this option to you but currently do not have an API publicly available yet. as we get closer to releasing Discussions more publicly i'll keep you posted on an update here!

**Reply by rhukster** [reply]
+1 for import from Discourse to Github Discussions.

**Reply by johnknapp** [reply]
Google Groups would be another sensible import source. So yes, please keep us updated here!

**Reply by sgallou** [reply]
From phpBB too !

**Reply by djipco** [reply]
I don't think you need to provide an importer for every single platform out there. Just having a standard format for import would be awesome. Personally, all I need is an archive of previous discussions because it still has value. The ability to import user profiles, preferences and such things would be nice but it is much less important than the actual messages themselves.

**Reply by bruceva** [reply]
We are hoping to import discussions from a Jive SBS database. An API to ease this process would be extremely helpful!

**Reply by gezelter** [reply]
Importing from mailman would be a great addition

**Reply by stevengj** [reply]
Just importing from mbox format would allow you to support a lot of different platforms…

**Reply by exportgenius-web** [reply]
I can suggest a platform where [import data](https://www.exportgenius.in) can be found here

**Reply by git-n-pissed** [reply]
@becca Any update on this?

**Reply by becca** [reply]
I no longer work at GitHub @git-n-pissed. Hopefully someone else on the team there can keep you posted on updates. 🙏

**Reply by JeneJhon** [reply]
> 

I also need this. Kindly update on a regular basis.Thanks
[AlightMotion](https://almotion.net/)

**Comment by junghans** [comment]
There is https://github.com/jeremyevans/gg2ghd, which seems to allow a rudimentary Google Group import.

**Comment by davidgiven** [comment]
I think emails imported by gg2ghd end up all being owned by whoever ran the script, which isn't very user friendly. It'd be nice to preserve the threading, too...

**Reply by Encroach11** [reply]
Yes but can apple or whoever ph me please

**Comment by Encroach11** [comment]
Apple said they would call me coz those guys might or obviously do already have my address w we
